gatsby-node.js

```diff
@@ -1,5 +1,8 @@
-exports.modifyWebpackConfig = function(webpack, options) {
-  webpack.config.loader('url-loader', {
+exports.modifyWebpackConfig = ({ config }, options) => {
+  const { plugins, dir, ...svgrOptions } = options;
+
+  // remove gatsby's url-loader loader for svgs
+  config.loader('url-loader', {
     test: /\.(jpg|jpeg|png|gif|mp4|webm|wav|mp3|m4a|aac|oga)(\?.*)?$/,
```

Comment:
Not really specific to this PR, but is there maybe a way to get this regex from Gatsby, then `replace` `|svg` so there's only a single source of truth?

Reply:
Ok I'll look into this. Maybe webpack-configurator's ```config.merge``` can help with this.

Reply:
I think we can do ```config.removeLoader('url-loader')``` to make sure gatsby's loader isn't included in the config. See [here](https://github.com/lewie9021/webpack-configurator#configremoveloaderkey) and  [here](https://github.com/gatsbyjs/gatsby/blob/0c1bfef8aa10bd225bda7aac628917507a216fc7/packages/gatsby/src/utils/webpack.config.js#L397)